server.js

```diff
@@ -57,9 +56,12 @@ app.prepare().then(() => {
       }
 
       // Custom route for all docs
-      app.render(req, res, '/doc', query)
+      app.render(req, res, '/doc', parseQuery(queryStr))
```

Comment:
was there a bug before? not sure I get this change

Reply:
minor refactor: it was calling `url.parse` twice because the first one sent `true` as the 2nd param to make the `query` part of the returned value into an object, but we could simply reuse that one call and parse just the query from string to query using `querystring.parse` (used internally in `url.parse` anyway). Same result mentioned in https://github.com/iterative/dvc.org/pull/973#pullrequestreview-353231202 above

Reply:
don't get it, sorry :(

so, `queryStr` has the same value as `query`? not sure still why do we call `parseQuery` on top of it?

Reply:
> queryStr has the same value as query

No, `queryStr` is a string, `query` is an object. `query`  = `parseQuery(queryStr)`

> See also my previous https://github.com/iterative/dvc.org/pull/973#pullrequestreview-353231202 above.

Reply:
k, I see, thanks @jorgeorpinel !